src/client/app/+about/components/about.component.spec.ts

```diff
@@ -2,25 +2,29 @@ import {
   TestComponentBuilder,
   describe,
   expect,
-  injectAsync,
-  it
-} from 'angular2/testing';
+  inject,
+  it,
+  AsyncTestCompleter
+} from 'angular2/testing_internal';
 import {Component} from 'angular2/core';
 import {DOM} from 'angular2/src/platform/dom/dom_adapter';
 import {AboutComponent} from './about.component';
 
 export function main() {
   describe('About component', () => {
+
+
     it('should work',
-      injectAsync([TestComponentBuilder], (tcb: TestComponentBuilder) => {
-        return tcb.createAsync(TestComponent)
+      inject([TestComponentBuilder, AsyncTestCompleter], (tcb: TestComponentBuilder, async: AsyncTestCompleter) => {
```

Comment:
I thought this too, but it fails with 

```
TypeError: undefined is not a constructor (evaluating 'new AsyncTestZoneSpec(finishCallback, failCallback, testName)') (line 98)
```

So i refered to the source to see how its implemented in the angular source